src/main/java/file/FileManager.java

```diff
@@ -0,0 +1,110 @@
+package file;
+
+import task.*;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class FileManager {
+
+    private static Path taskDataPath = Paths.get(System.getProperty("user.dir"), "data", "tasks.txt");
```

Comment:
Maybe consider replacing the `tasks.txt` portion with a constant, to eliminate magic literals.